src/test/java/org/broadinstitute/hellbender/tools/walkers/rnaseq/OverhangFixingManagerUnitTest.java

```diff
@@ -271,4 +273,12 @@ public void testUnalignedReadNotClearingReads() {
         Assert.assertEquals(manager.getReadsInQueueForTesting().size(), 1);
     }
 
+    @Test
+    public void testReadWithDeletionsWindowSpanning() {
+        final OverhangFixingManager manager = new OverhangFixingManager(getHG19Header(), null, hg19GenomeLocParser, hg19ReferenceReader, 100, 100, 30, false, true);
+        OverhangFixingManager.Splice splice = manager.addSplicePosition("1",6816, 11247);
+        GATKRead read = ArtificialReadUtils.createArtificialRead(hg19Header,"read1",0,11244, new byte[100], new byte[100],"97S2M18D1M");
```

Comment:
I hate to make this particular nitpick, but could you fix the spaces in this list of parameters, they're really atrociously inconsistent.